### ISSUE TITLE
is.gd blocked by dns provider

### ISSUE THREAD

**Issue by rpoulin** [dmpop/mejiro#15]
Comodo DNS is blocking is.gd as "malicious" which causes the call to curl to hang the application.

![screenshot_20160513_201459](https://cloud.githubusercontent.com/assets/2037530/15265714/845b432e-1949-11e6-9008-779eb0a3d12d.png)
  
The short link feature is a good idea as is and should be preserved but, in this case, it gets in the way. Any ideas?

**Comment by dmpop** [comment]
Thank you for your comment and proposal. It would be absolutely
fantastic if you could implement both options, so it's possible to
enable the URL shortening option and specify the desired service.

**Comment by rpoulin** [comment]
Hi Dmitri,

Can you point me to the documentation that would explain your decision to
pull out the photos/tims directory creation functionality?  You noted
security but I could not find any information about it except for the
default security descriptor of the mkdir statement (0777) which you
countered by issuing a chmod statement immediately after the creation.   I
am asking because the missing tims directory is an issue if you install the
application from scratch. The script does not test for the presence of the
tims directory. Can you explain what your thoughts were on this?

Best,
Ron

**Comment by dmpop** [comment]
Hi Ron,

The way the photos/tims directory creation functionality was implemented allowed anyone to create directories by simply adding arbitrary names to the base URL. For example, pointing the browser to http://domain.com/mejiro/index.php?d=/foo would create the *foo* directory if it didn't already exist. I didn't consider that an issue until I recently found my own Mejiro installation littered with directories like *xxfxfxfx* and *asadasad*. So I've removed this functionality to prevent this from happening again. I updated to the script, so it now tests for the presence of the *tims* directory. Thanks the pointer!

Best,
Dmitri
